material_components_ios.bzl

```diff
@@ -253,20 +255,48 @@ def mdc_snapshot_test(
       size = size,
       **kwargs)
 
+def mdc_ci_config_setting():
+    """Config setting for mdc continuous integration, e.g. --define ci_mode=kokoro"""
+    native.config_setting(
+        name = "kokoro",
+        values = {"define": "ci_mode=kokoro"},
+    )
+    native.config_setting(
+        name = "autobot",
+        values = {"define": "ci_mode=autobot"},
+    )
+
+def mdc_unit_test_extension_runners_select():
+    """Config select for unit test extension used by mdc continuous integration"""
+    mdc_ci_config_setting()
+    return select({
+        ":kokoro": KOKORO_EXTENSION_IOS_RUNNER_TARGET,
+        ":autobot": AUTOBOT_EXTENSION_IOS_RUNNER_TARGET,
+        "//conditions:default": KOKORO_EXTENSION_IOS_RUNNER_TARGET, 
+    })
```

Comment:
This select statement can then be moved down below to mdc_unit_test_suite.

Reply:
The function was made to prepare for a future `mdc_snapshot_test_extension_runners_select` function to make it clear what runner is used. I can move it into `mdc_unit_test_suite` using a variable if that looks more clear. How do you feel?

Reply:
Can we fold this in to mdc_unit_test_suite so that we don't have another top-level method here?

Reply:
Sure. Addressed in db14099a620a16a49354184cf31d8a91ce0a85b3.